go.mod

```diff
@@ -1,79 +1,67 @@
-module github.com/cosmos/interchain-security/v4
+module github.com/cosmos/interchain-security/v5
 
-go 1.21.1
+go 1.22.2
 
 require (
 	cosmossdk.io/errors v1.0.1
 	cosmossdk.io/math v1.3.0
-	github.com/cometbft/cometbft v0.37.6
-	github.com/cometbft/cometbft-db v0.11.0
-	github.com/cosmos/cosmos-sdk v0.47.12
-	github.com/cosmos/gogoproto v1.4.10
-	github.com/cosmos/ibc-go/v7 v7.6.0
+	github.com/cometbft/cometbft v0.38.7
```

Comment:
Should we use [v0.38.8](https://github.com/cometbft/cometbft/releases/tag/v0.38.8) that includes 2 medium-security bug fixes and performance improvements?

Reply:
pls, note that there' was a change in v0.38.8 which got reverted in v0.38.9 [see here](https://github.com/cometbft/cometbft/blob/v0.38.9/CHANGELOG.md#v0389) and the Mempool API got back to the same as in v0.38.7.
have the impression that v0.38.8 is not the best choice. Either v0.38.7 or v0.38.9 (SDK v0.50.7 uses v0.38.7).

Reply:
Using cometBFT v38.9 sounds good to me. Thanks for looking at this!